pkg/encrypt.go

```diff
@@ -54,7 +54,7 @@ func HandleEncryptionRequest(ctx context.Context, name, location string, dataDel
 		log.Warn().Msg("encrypt failed")
 		_, ddsErr := dataDeliveryStatusClient.Error("errored", name, err.Error())
 		if ddsErr != nil {
-			log.Err(err).Msgf("Updating data delivery status to 'errored' failed: %s", ddsErr.Error())
+			log.Error().Msgf("Updating data delivery status to 'errored' failed: %s", ddsErr.Error())
```

Comment:
Looking at the package, both log.Error and log.Err both output logs in the "error" level. It seems like there aren't many differences apart from log.Err() taking an argument of type "error".

I've found that the error logs they generate look a bit different but to standardise things, I chose to use log.Error(). 
I've seen instances where log.Err(err) is used where the error message is displayed twice unnecessarily like this line.

Example of output:
_Using log.Error().Msgf("Updating data delivery status to 'errored' failed: %s", ddsErr.Error())_
`{"level":"error","time":"2023-09-03T19:59:00+01:00","message":"Updating data delivery status to 'errored' failed: '<ERROR>'"}`

_log.Err(err).Msgf("Updating data delivery status to 'errored' failed: %s", ddsErr.Error())_
`{"level":"error","error":"<ERROR>","time":"2023-09-03T20:00:47+01:00","message":"Updating data delivery status to 'errored' failed: '<ERROR>'"}`

Reply:
Thinking about it now, it might be better to use log.Err() instead so that we are able to separate and get the error message.